docker-compose/owls/owls.env

```diff
@@ -0,0 +1,40 @@
+RUN_CHOWN=true
+TEMPLATE_CONFIG=true
+SELFSIGNED_CERTS=true
+
+#ROOTCA=$OWLS_ROOT/certs/root.pem
```

Comment:
Can we remove these values instead of commenting them out or have you kept these for a reason?

Reply:
I added them for an overview of all supported configuration options and their default values.